src/containers/orders/cart/filled-cart/filled-cart.js

```diff
@@ -1,24 +1,62 @@
-import React, { useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import { Link } from 'react-router-dom';
-import KeyboardBackspaceIcon from '@material-ui/icons/KeyboardBackspace';
 
 import OrderTable from '../../order/order-table';
 import Modal from '../../../../components/modal';
 import { MODAL_DELETE_MESSAGES } from '../../../../translations/modal.translations';
 import { removeItemFromCart } from '../../../../redux/cart/cart.actions';
 import CartItem from '../cart-item';
 import { useStyles } from './filled-cart.styles';
+import { Button } from '@material-ui/core';
+import { CART_BUTTON_TITLES } from '../../../../translations/cart.translations';
+import SimilarProducts from '../../../../pages/product-details/similar-products/similar-products';
+import { faDollarSign, faHryvnia } from '@fortawesome/free-solid-svg-icons';
+import {
+  getFiltredProducts,
+  getProduct,
+  setCategoryFilter
+} from '../../../../redux/products/products.actions';
 
-const FilledCart = ({ items }) => {
+const FilledCart = ({ items, categories }) => {
   const [modalVisibility, setModalVisibility] = useState(false);
   const [modalItem, setModalItem] = useState({});
   const styles = useStyles();
   const dispatch = useDispatch();
-  const { language, currency } = useSelector(({ Language, Currency }) => ({
-    language: Language.language,
-    currency: Currency.currency
-  }));
+  const { language, goods, currency } = useSelector(
+    ({
+      Language,
+      Currency,
+      Products: { product, productLoading, productToSend }
+    }) => ({
+      language: Language.language,
+      currency: Currency.currency,
+      isLoading: productLoading,
```

Comment:
Why do you return unused fields like `isLoading` and `productToSend`?

Reply:
Fixed